packages/polyfills/src/config.ts

```diff
@@ -0,0 +1,51 @@
+import {isSupported} from 'caniuse-api';
+
+export interface PolyfillDescriptor {
+  supportsNode: boolean;
+  featureTest?: string;
+}
+
+export const polyfills: {[polyfill: string]: PolyfillDescriptor} = {
+  baseline: {
+    supportsNode: true,
+  },
+  fetch: {
+    supportsNode: true,
+    featureTest: 'fetch',
+  },
+  intl: {
+    supportsNode: false,
+    featureTest: 'internationalization-plural-rul',
+  },
+  url: {
+    supportsNode: true,
+    featureTest: 'urlsearchparams',
+  },
+};
+
+export function mappedPolyfillsForEnv(browser: 'node' | string | string[]) {
+  const mappedPolyfills = {};
+
+  Object.entries(polyfills).forEach(
+    ([polyfill, {supportsNode, featureTest}]) => {
```

Comment:
@lemonmade I don't think this can me `reduce`d since the result is an object not an array.